style.css

```diff
@@ -25,10 +26,72 @@
     align-items: center;
     width: 100%;
     height: 50px;
+    position: relative;
   }
 
   .top-nav {
-    display: none;
+    width: 100%;
+    height: 720px;
+    background-color: rgb(247, 247, 249);
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 999;
+    padding: 28px;
+    transform: translateX(130%);
+    transition: all 1s ease-in-out;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+  }
```

Comment:
- [ ] kindly make a top-nav `display: none;` and when to click in hamburger menu make it `display: block;` , so this prevents the design issue like the following image when scrolling to right, fix this in mobile version and desktop version.
>![image](https://user-images.githubusercontent.com/53402600/188633175-3db31c0a-1c76-4b2c-9040-10baa4ebd461.png)